.size-snapshot.json

```diff
@@ -1,21 +1,21 @@
 {
   "dist/react-beautiful-dnd.js": {
-    "bundled": 372683,
-    "minified": 142833,
-    "gzipped": 40327
+    "bundled": 399623,
+    "minified": 149169,
+    "gzipped": 42431
   },
   "dist/react-beautiful-dnd.min.js": {
-    "bundled": 334038,
-    "minified": 126662,
-    "gzipped": 35556
+    "bundled": 360109,
+    "minified": 132835,
+    "gzipped": 37619
   },
   "dist/react-beautiful-dnd.esm.js": {
```

Comment:
our esm builds are looking better for moving to invariant rather than throwing, but otherwise we took a huge hit!

Reply:
Here is the reason for using invariant:

```js
 * The invariant message will be stripped in production, but the invariant
 * will remain to ensure logic does not differ in production.
```

Flow understands that invariant will throw if the condition is not met